### PR TITLE
Add language support for user creation in api

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]
Code makes sense :+1:

**Review by rullzer — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Sorry for the failed jobs ... I checked the wrong page. @skjnldsv will do a PR to fix the failures.

File: apps/provisioning_api/lib/Controller/UsersController.php [hunk 1]
```diff
@@ -279,6 +280,10 @@ public function addUser(string $userid,
 				$this->editUser($userid, 'quota', $quota);
 			}
 
+			if ($language !== '') {
+				$this->editUser($userid, 'language', $language);
```
Comment (nickvergessen) [inline apps/provisioning_api/lib/Controller/UsersController.php]: this should also be used in the "new user mail" ?!

Comment (rullzer) [reply apps/provisioning_api/lib/Controller/UsersController.php]: That when you don't specifiy a password. So the new user-email is send. It should be in this language.

Comment (MorrisJobke) [reply apps/provisioning_api/lib/Controller/UsersController.php]: Ah - got it 👍

**Comment by MorrisJobke** [comment]
Let's revert it for now.

**Pull request merged**